users/vieting/experiments/switchboard/ctc/feat/network_helpers/perturbation.py

```diff
@@ -69,7 +71,7 @@ def __init__(
         self._perturbations = []
         self.non_linearity = non_linearity
         if preemphasis:
-            self._perturbations.append(functools.partial(self.preemphasis, factor=PerturbationFactor(**preemphasis)))
+            self._perturbations.append(functools.partial(self.preemphasis, preemphasis_args=preemphasis))
```

Comment:
You can extend `PerturbationFactor` with a default value, that would be cleaner than the current solution.

Reply:
functions that need more than these three get the dict directly.

Reply:
But wouldn't it be nicer to add an optional default value?

Reply:
I will do this